### DISCUSSION TITLE
Duda con práctica JS (objetos en la API)

### DISCUSSION THREAD

**Discussion by KevinSalgado** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#396] [Q&A] [answered]
Hola! Estoy teniendo unos problemas a la hora de consultar las estadísticas de los pokemones en la API, ya que debo usar números para poder entrar a estos. 
![image](https://user-images.githubusercontent.com/99103497/158097327-d7149ef8-ccb7-445c-8d53-f5a78c86a34e.png)
Como podrán ver, para poder acceder a las estadísticas debo hacer algo como lo siguiente: let pokeHp = data.stats.0;
![image](https://user-images.githubusercontent.com/99103497/158097457-5a914d17-04cb-4f1b-aa84-21f372ffe167.png)
Pero me marca error al querer poner un número, así llevo varias horas intentado buscar una forma de entrar a los datos de las estadísticas, pero no he podido.

**Comment by AlejandroMorales-s** [comment, marked as answer]
Holaa, yo hice esto para acceder a las estadísticas del pokemón 
![Imagen1](https://user-images.githubusercontent.com/77386102/158097932-aef10417-3f8a-4824-aacb-3dd4312e9d8a.png)

**Reply by KevinSalgado** [reply]
Muchas gracias

**Reply by AngelCruzO** [reply]
tambien pueden desplegarlas con un for para optimizar codigo

**Reply by jricardocam** [reply]
Hola
De que manera puedo desplegarlas? ya que al ejecutar mi codigo solo toma el ultimo valor para todas las estadisticas
gracias

**Reply by AngelCruzO** [reply]
Yo lo que hice fue antes de que me retorne el json, volver a asignarle un valor vacío, para resetear el valor. Stats.innerHTML = ""
